first_year_scraper/main.py

```diff
@@ -0,0 +1,22 @@
+import csv
+import excel_parser
+import excel_formatter
+import restructure
+
+filename = input("Enter the pdf file name: ")
```

Comment:
A command-line argument would be cleaner, I guess, or if it is purposely done, please tell me. I may be wrong.

Reply:
@sriyash421 this has been mentioned above in the comment of @icyflame. If you don't mind we can keep this in one conversation on the top and resolve this!😀